diff --git a/dist/esm/server/load-components.js b/dist/esm/server/load-components.js
index 6cfc939c4ca337eeea376b7a172040697bf58b24..1044959d1a8927649d602aeab777cecc49488ea0 100644
--- a/dist/esm/server/load-components.js
+++ b/dist/esm/server/load-components.js
@@ -87,7 +87,7 @@ async function loadComponentsImpl({ distDir, page, isAppPath, isDev, sriEnabled
         loadManifestWithRetries(join(distDir, BUILD_MANIFEST), manifestLoadAttempts),
         tryLoadManifestWithRetries(reactLoadableManifestPath, manifestLoadAttempts),
         // This manifest will only exist in Pages dir && Production && Webpack.
-        isAppPath || process.env.TURBOPACK ? undefined : loadManifestWithRetries(join(distDir, `${DYNAMIC_CSS_MANIFEST}.json`), manifestLoadAttempts).catch(()=>undefined),
+        isAppPath || isDev || process.env.TURBOPACK ? undefined : loadManifestWithRetries(join(distDir, `${DYNAMIC_CSS_MANIFEST}.json`), manifestLoadAttempts).catch(()=>undefined),
         isAppPath && hasClientManifest ? tryLoadClientReferenceManifest(join(distDir, 'server', 'app', page.replace(/%5F/g, '_') + '_' + CLIENT_REFERENCE_MANIFEST + '.js'), page.replace(/%5F/g, '_'), manifestLoadAttempts) : undefined,
         isAppPath ? loadManifestWithRetries(join(distDir, 'server', SERVER_REFERENCE_MANIFEST + '.json'), manifestLoadAttempts).catch(()=>null) : null,
         sriEnabled ? loadManifestWithRetries(join(distDir, 'server', SUBRESOURCE_INTEGRITY_MANIFEST + '.json')).catch(()=>undefined) : undefined
diff --git a/dist/server/load-components.js b/dist/server/load-components.js
index a25b52ea3919be6d175bb5cbf773efb984a719c0..5b2ac41a60d7f3f8ba5e0ed645ef05568cd519f7 100644
--- a/dist/server/load-components.js
+++ b/dist/server/load-components.js
@@ -111,7 +111,7 @@ async function loadComponentsImpl({ distDir, page, isAppPath, isDev, sriEnabled
         loadManifestWithRetries((0, _path.join)(distDir, _constants.BUILD_MANIFEST), manifestLoadAttempts),
         tryLoadManifestWithRetries(reactLoadableManifestPath, manifestLoadAttempts),
         // This manifest will only exist in Pages dir && Production && Webpack.
-        isAppPath || process.env.TURBOPACK ? undefined : loadManifestWithRetries((0, _path.join)(distDir, `${_constants.DYNAMIC_CSS_MANIFEST}.json`), manifestLoadAttempts).catch(()=>undefined),
+        isAppPath || isDev || process.env.TURBOPACK ? undefined : loadManifestWithRetries((0, _path.join)(distDir, `${_constants.DYNAMIC_CSS_MANIFEST}.json`), manifestLoadAttempts).catch(()=>undefined),
         isAppPath && hasClientManifest ? tryLoadClientReferenceManifest((0, _path.join)(distDir, 'server', 'app', page.replace(/%5F/g, '_') + '_' + _constants.CLIENT_REFERENCE_MANIFEST + '.js'), page.replace(/%5F/g, '_'), manifestLoadAttempts) : undefined,
         isAppPath ? loadManifestWithRetries((0, _path.join)(distDir, 'server', _constants.SERVER_REFERENCE_MANIFEST + '.json'), manifestLoadAttempts).catch(()=>null) : null,
         sriEnabled ? loadManifestWithRetries((0, _path.join)(distDir, 'server', _constants.SUBRESOURCE_INTEGRITY_MANIFEST + '.json')).catch(()=>undefined) : undefined
